lib/hooks.js

```diff
@@ -36,7 +36,17 @@ Hookable.afterUpdate = null;
 Hookable.beforeDestroy = null;
 Hookable.afterDestroy = null;
 
-// TODO: Evaluate https://github.com/bnoguchi/hooks-js/
+/**
+ * @deprecated
+ * Setup a Model-based hook to trigger when the specified action occurs.
+ * The trigger is broken up into three segments: `beforeHook`, `work` and
+ * `afterHook`.
+ * @param {string} actionName The name of the action that triggers the hook.
+ * @param {Function} work The 2nd phase of the trigger.
+ * @param {*} data The value(s) to provide to the 1st phase (`beforeHook`) call.
+ * @callback
```

Comment:
It's a special decorator that marks the function as a callback.
For humans, it's superfluous, but it tells parsers that the specific parameter at that index is a callback (so if the param was named `asdf`, it would still know that it's a callback).